Riot/Modules/Home/AllChats/AllChatsFilterOptions.swift

```diff
@@ -40,7 +40,18 @@ class AllChatsFilterOptions: NSObject {
                 AllChatsLayoutSettingsManager.shared.activeFilters = []
                 return
             }
-         
+            
+            switch filter {
+            case .all:
+                Analytics.shared.trackInteraction(.allChatsFilterAll)
+            case .favourites:
+                Analytics.shared.trackInteraction(.allChatsFilterFavourites)
+            case .people:
+                Analytics.shared.trackInteraction(.allChatsFilterPeople)
+            case .unreads:
+                Analytics.shared.trackInteraction(.allChatsFilterUnreads)
+            default: break
```

Comment:
Let's also get rid of this default

Reply:
Actually it's not exhaustive. We have a `rooms` case that we don't use anymore for the moment.